linkerd2.yaml

```
@@ -29,6 +29,10 @@ pipeline:
       repository: https://github.com/linkerd/linkerd2/
       tag: edge-${{package.version}}
 
+  - uses: go/bump
+    with:
+      deps: golang.org/x/crypto@v0.31.0 idna@1.0.0
```

```suggestion
      deps: golang.org/x/crypto@v0.31.0
```
idna is a rust crate and can be removed from here https://docs.rs/idna/latest/idna/